independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/ObserverInfo.java

```diff
@@ -143,6 +145,17 @@ MethodParameterInfo initEventParam(MethodInfo observerMethod) {
                     LOGGER.warnf(transactionalObserverWarning, clazz, observerMethod.name(), during.asEnum());
                 }
             }
+            if (annotation.name().equals(DotNames.create(Initialized.class))
```

Comment:
I think that we should only log this message for app classes, ie. classes that are under the control of the user. We do this for private members as well.

Also I think that we should handle this in a build step in the `ArcProcessor`.